### DISCUSSION TITLE
Is it possible to sort tabs ?

### DISCUSSION THREAD

**Discussion by dubinfo** [PHPOffice/PhpSpreadsheet#4045] [General]
Hello,

Is it possible to sort the tabs of my Excel workbook once it is generated?   

PhpSpreadsheet Version : 1.18 Php 8.1 Laravel 8.75

Thank you in advance for your help.

David

**Comment by jr212** [comment]
Google found https://stackoverflow.com/questions/78478646/how-to-sort-worksheets-by-name-in-a-phpspreadsheet-workbook-without-disrupting-s

**Comment by oleibman** [comment]
```php
        $titleArray = $spreadsheet->getSheetNames();
        sort($titleArray);
        $index = -1;
        foreach ($titleArray as $title) {
            ++$index;
            $spreadsheet->setIndexByName($title, $index);
        }
```

**Comment by oleibman** [comment]
Don't see a way to "mark as answer", but, 3+ months after the last message, no reason to think it isn't right. Closing discussion.
